app/src/app/data.service.ts

```diff
@@ -6,17 +6,22 @@ import { Capability } from './capability';
 import { Family } from './family';
 import { Role } from './role';
 import { User } from './user';
+import { CapabilityLead } from './capabilityLead';
+
 
 @Injectable({
   providedIn: 'root'
 })
 export class DataService {
 
   public user: User;
+  public role: Role;
+  public splittedResponsibilities: string[];
+  public splittedTraining: string[];
 
   constructor(private http: HttpClient) {
     this.getUser();
-  }
+   }
 
   public getUser(): void {
```

Comment:
refactor for methods to return Obeservables....
This method is clearly not been tested! refactor and ensure there are tests for these
I know its not part of you changes but still should be done